Makefile

```diff
@@ -58,19 +58,23 @@ $(ENV_DIR)/bin/python:
 $(ENV_DIR)/bin/coverage: $(ENV_DIR)/bin/python
 	$(ENV_DIR)/bin/pip install coverage
 
-.PHONY: build_common_egg
-build_common_egg: $(ENV_DIR)/bin/python setup.py libclearwaterutils.a
-	$(COMPILER_FLAGS) ${ENV_DIR}/bin/python setup.py bdist_egg -d $(EGG_DIR)
+.PHONY: build_common_wheel
```

Comment:
Is the naming right? It looks like this doesn't just build the wheel for the current dir, it also builds wheels for all of the dependencies.

Reply:
Correct. On reflection, I think it should just build the common wheel, and then we end up with a more straight forward approach.